### PR TITLE
Unify the bundling system of `blazor server/wasm and mvc`.

### LINKED ISSUE #19963: Create blazor `global js&css` at runtime.

### PR DESCRIPTION
**Pull request by maliming**
Resolves #19963

After this PR, we don't need the `abp bundle` command to manually create the `global.js` and `global.css` files.

---

### How it works?

https://github.com/abpframework/abp/blob/dev/docs/en/Community-Articles/2024-11-25-Global-Assets/POST.md

---

LeptonX needs this PR change, and the template project also needs the LeptonX PR change.

We need to release the nightly build packages of abp and leptonx.

---

abp nightly build package: `9.1.0-preview20241210`
leptonx nightly build package: `4.1.0-preview20241210`

--- 

ABP and LeptonX are ready to merge, waiting for Volo to change and test.

https://github.com/volosoft/lepton/pull/2291
https://github.com/volosoft/volo/pull/17925

### CODE REVIEW

File: templates/app-nolayers/aspnet-core/MyCompanyName.MyProjectName.Mvc.Mongo/MyCompanyName.MyProjectName.Mvc.Mongo.csproj [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <ProjectReference Include="..\..\..\..\framework\src\Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared\Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared.csproj" />
     <ProjectReference Include="..\..\..\..\framework\src\Volo.Abp.AutoMapper\Volo.Abp.AutoMapper.csproj" />
     <!-- </TEMPLATE-REMOVE> -->
-    <PackageReference Include="Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite" Version="3.2.0" />
+    <ProjectReference Include="..\..\..\..\..\..\lepton\aspnet-core\abp\src\Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite\Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite.csproj" />
```
Comment (EngincanV) [inline templates/app-nolayers/aspnet-core/MyCompanyName.MyProjectName.Mvc.Mongo/MyCompanyName.MyProjectName.Mvc.Mongo.csproj]: This might not be replaced by the ABP CLI? I guess this is added for testing purposes and we should revert this change, right? @maliming

**Review by EngincanV — APPROVED** [review]

File: templates/app-nolayers/aspnet-core/MyCompanyName.MyProjectName.Mvc.Mongo/MyCompanyName.MyProjectName.Mvc.Mongo.csproj (hunk 1 above)
Comment (maliming) [reply templates/app-nolayers/aspnet-core/MyCompanyName.MyProjectName.Mvc.Mongo/MyCompanyName.MyProjectName.Mvc.Mongo.csproj]: You are right, this is the test code. I have reverted the code.

**Review by enisn — APPROVED** [review]

**Review by realLiangshiwei — APPROVED** [review]

**Review by enisn — APPROVED** [review]
I just made my final checks and I think it's ready to go now. Should we merge it now?

**Comment by maliming** [comment]
> I just made my final checks and I think it's ready to go now. Should we merge it now?

@realLiangshiwei is testing the Volo side. We will merge once all tests pass.
